--- a/cinelerra-5.1/thirdparty/src/x265_4.0.patch2
+++ b/cinelerra-5.1/thirdparty/src/x265_4.0.patch2
@@ -1,7 +1,12 @@
 --- /dev/null	2020-03-14 06:02:18.586124011 +0300
 +++ ./Makefile	2020-03-18 00:04:59.388807329 +0300
-@@ -0,0 +1,4 @@
-+#$(shell cd build/linux ; ./multilib.sh)
+@@ -0,0 +1,9 @@
 +.NOTPARALLEL:
-+all:
-+	$(shell ./multilib.sh ; cp 8bit/libx265.a . ; cp 8bit/x265.pc . ; cp 8bit/x265_config.h .)
++all: multilib lib_copy
++
++multilib:
++		./multilib.sh
++
++lib_copy:
++		cp 8bit/libx265.a .
++		cp 8bit/x265_config.h .
